src/content/reference/hi/p5/lightFalloff.mdx

```diff
@@ -1,35 +1,18 @@
 ---
 title: lightFalloff
 module: 3D
-submodule: Lights
+submodule: रोशनी
```

Comment:
use "प्रकाश" for lights since you have used that previously so it will be consistent

Reply:
Done